src/api/apiAxios.ts

```diff
@@ -24,6 +24,9 @@ export const newApiAxios = (base: 'server' | 'client') => {
 
       debugServer('[SERVER API]', method?.toUpperCase(), url && baseURL ? new URL(url, baseURL).href : url)
 
+      config.headers['X-Forwarded-Host'] = 'test.strom.sk'
+      config.headers['X-Forwarded-Proto'] = 'https'
```

Comment:
neviem, akymi env varmi by som to riesil a preco... lokalne je to zda sa jedno. asi doladime na env neskor, ked to pojde na strom.sk, abo tak